Source/WebCore/svg/SVGPolyElement.h

```diff
@@ -33,8 +33,13 @@ class SVGPolyElement : public SVGGeometryElement {
 public:
     const SVGPointList& points() const { return m_points->currentValue(); }
 
+    // per SVG 2 https://svgwg.org/svg2-draft/shapes.html#InterfaceSVGAnimatedPoints
+    // The points and animatedPoints IDL attributes represent the current
+    // non-animated value of the reflected attribute.
+    // On getting points or animatedPoints, an SVGPointList object is returned that
+    // reflects the base value of the reflected attribute.
     SVGPointList& points() { return m_points->baseVal(); }
-    SVGPointList& animatedPoints() { return *m_points->animVal(); }
+    SVGPointList& animatedPoints() { return m_points->baseVal(); }
```

Comment:
I think this change is wrong. `SVGPolyElement` does not have a member of type `SVGAnimatedPointList`. It has the `baseVal` and the `animVal` of the points flattened to two members `points` and `animatedPoints`. If you make this change, there will be no access to the `animVal` of the points ever. See my comment here: https://github.com/w3c/svgwg/issues/706#issuecomment-2985118963

Reply:
Thanks a lot @shallawa. I will close the PR.